pkg/cmd/server/api/types.go

```diff
@@ -65,6 +65,8 @@ type MasterConfig struct {
 	ImageConfig ImageConfig
 
 	PolicyConfig PolicyConfig
+
+	SchedulerConfigFile string
```

Comment:
If this is only used by kubernetes, it should be moved to the `KubernetesMasterConfig` type.